### ISSUE TITLE
Live Server: unable to select all inside of the preview

### ISSUE THREAD

**Issue by misolori** [microsoft/vscode#125955]
![image](https://user-images.githubusercontent.com/35271042/121582585-07788f00-c9e4-11eb-8b39-dabed063779d.png)

I have an input field in one of my HTML pages and often will select all the text to clear it but unfortunately I can't do that with the Live Server preview.

**Comment by andreamah** [comment]
Hmm that's interesting, I can't replicate this on windows. Are you on mac?

**Comment by misolori** [comment]
Yes, I'm on a mac. Let me test this on Windows to see if i get the same thing. Also to clarify, I used the keyboard shortcut to select all.

**Comment by andreamah** [comment]
I tried this in Linux and seemed that the select-all shortcut works?
[![Image from Gyazo](https://i.gyazo.com/f13bf18a09cc47840f80987401fc116c.gif)](https://gyazo.com/f13bf18a09cc47840f80987401fc116c)

When you have a sec, can you try selecting all on the current release? https://github.com/microsoft/vscode-liveserver/releases/tag/0.0.5

**Comment by misolori** [comment]
Tested this on Windows and it works fine. I also removed all of my keybindings that referenced `cmd+a` on my Mac and it still repros (even in a fresh user data dir). Also tested a basic text input and it repros:

<img width="976" alt="CleanShot 2021-07-09 at 09 01 07@2x" src="https://user-images.githubusercontent.com/35271042/125106345-62111380-e094-11eb-8821-9af1e5d6e2c4.png">

cc @deepak1556 @rzhao271 to see if this is Electron related

**Comment by misolori** [comment]
Worth mentioning that this repros in the simple browser as well:

<img width="623" alt="CleanShot 2021-07-09 at 09 04 34@2x" src="https://user-images.githubusercontent.com/35271042/125106648-b74d2500-e094-11eb-8fad-211885e1828b.png">
